docs/Chapter02-10-ScrubbingMetadata.md

```diff
@@ -0,0 +1,92 @@
+# Scrubbing Metadata from Files- A low-fi approach
+
+## Overview 
+Files such as Word documents and JPEG images usually contain information about the systems used to create them. This information, commonly referred to as metadata, could inadvertently reveal personally identifiable details about your sources to anyone given access to the files. Removing metadata from files provided by sources before sharing or publishing them is critical for anonymous source protection.
+
+This lesson plan covers what you'll need to know to find file metadata in Word documents and JPEG files, and how you can share or publish content from these files without exposing metadata.
+
+## About This Lesson Plan
+
+**Review date:** June 5 2017  
+**Lesson duration:** 45 minutes  
+**What will participants learn?**  
+Basic understanding of what metadata is and basic techniques to "clean" common file formats before publishing or sharing. This session does \*not\* cover myriad other types of metadata, most notably the metadata created by journalists and sources when they browse the web, use their mobile phones, or communicate using VoIP, chat, email, etc.
+
+**What materials will participants need?**  
+Email access, a laptop with MS Word installed, access to printer and scanner 
+
+**What materials will the instructor need?**  
+Sample document (like a PDF or Word document) and sample image (JPEG or similar format), both with identifying metadata included—such as the name of the Microsoft license-holder, geotag/location data, etc.
+
+**How should the instructor prepare?**   
+Review lesson plan, and share example documents from ScrubbingMetadata folder with participants.
+
+## Lesson Plan
+
+### Icebreakers/discussion 
+
+Did you know that sharing and publishing files can burn your sources?
+
+Example: Vice and John McAfee: [*https://www.vice.com/en\_us/article/john-mcafee-bath-salts-belize-murder-fugitive-gregory-faull*](https://www.vice.com/en_us/article/john-mcafee-bath-salts-belize-murder-fugitive-gregory-faull) - When Vice published the story, the photos still had location data included and basically burned their source.
+
+Example: [*How The Intercept Outed Reality Winner*](http://blog.erratasec.com/2017/06/how-intercept-outed-reality-winner.html) - Reality Winner used traditional mail, not the Internet, to share documents with the Intercept, but was caught due to both operational security mistakes and metadata included in the printed files she shared.
```

Comment:
I find myself wondering if we ought to call out the whole color-printer watermark thing here in a little more detail. I know it's mentioned later on in the lesson, but to me it's such a crazy thing that few people participating in a lesson like this will have ever heard of. It might be worth raising more detail here just for that impact—things that you _never even imagined_ can burn your sources.

Reply:
Just thinking that's the kind of example that could get people really paying attention.

Reply:
I guess - one problem this lesson has is that it doesn't really get into metadata that's there for adversarial reasons. Stuff like yellow dots on printouts are fairly straightforward, typographical changes and intentional misspellings are examples of for lack of a better term steganographic metadata and are very much next-level, and then there's things like matching file creases on scanned paper docs, which isn't metadata per se but can still be used to identify a document's source. I can highlight them a bit more but figuring out how to identify and defend against them would add another level of complexity. TBH I think it could be another lesson plan.